.github/workflows/pypi-release.yml

```diff
@@ -17,23 +17,24 @@ jobs:
       - uses: actions/checkout@v4
         with:
           submodules: "recursive"
-      # NOTE: enable later to build for aaarch64
-      # NOTE: this is not working yet, because of grpcio dependency
-      #- name: Set up QEMU
-      #  if: runner.os == 'Linux'
-      #  uses: docker/setup-qemu-action@v3
-      #  with:
-      #    platforms: all
+      - name: Set up QEMU
+        if: runner.os == 'Linux'
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: all
       - name: Install dependencies
-        run: python3 -m pip install --upgrade pybind11 setuptools wheel cibuildwheel==2.16.2
+        run: python3 -m pip install --upgrade pybind11 setuptools wheel cibuildwheel==2.3.1
```

Comment:
Is the constraint `cibuildwheel==2.3.1` just for build consistency, or is there a specific reason? Worth adding a comment.

Reply:
that's a mistake actually :)